packages/frontend-plugin-api/src/blueprints/SignInPageBlueprint.test.tsx

```diff
@@ -60,20 +62,11 @@ describe('SignInPageBlueprint', () => {
 
     const tester = createExtensionTester(extension);
 
-    expect(tester.data(SignInPageBlueprint.dataRefs.component)).toBeDefined();
+    const Element = tester.get(SignInPageBlueprint.dataRefs.component)!;
```

Comment:
The "!" seems wrong. Would it be possible for the tester to infer which dataRefs the subject being tested has. Something like:
```tsx
const Component = tester.subject.dataRefs.component;
```

Reply:
This got me thinking that we could probably type this better, so now we don't need the `!`. It will also complain if you try to to `.get` for a `dataRef` that the extension under test does not export.

We unfortunately don't know the `dataRefs` by any form of key once it's been made into an `ExtensionDefinition` that's why they're wrapped up on the blueprint.

Reply:
Thanks for explaining, the type looks great 💯